scripts/skills/effects/rf_bone_plating_effect.nut

```diff
@@ -0,0 +1,34 @@
+this.rf_bone_plating_effect <- ::inherit("scripts/skills/skill", {
```

Comment:
The vanilla item is called `bone_platings_upgrade` so we should follow that naming convention and rename the skill file as well as the images/brush to be plural i.e. `platings`.